### PR TITLE
chore: bump mockative to 1.4.1

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

bump mockative to 1.4.1

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by gongracr — APPROVED** [review]
LGTM. Great job @vitorhugods for chasing this sausage 🦅

File: logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/backup/RestoreWebBackupUseCaseTest.kt [hunk 1]
```diff
@@ -128,7 +128,7 @@ class RestoreWebBackupUseCaseTest {
         verify(arrangement.migrationDAO)
             .suspendFunction(arrangement.migrationDAO::insertConversation)
             .with(any())
-            .wasInvoked()
+            .wasNotInvoked()
```
Comment (ohassine) [inline logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/backup/RestoreWebBackupUseCaseTest.kt:131]: Is that correct you changed it from `wasInvoked` to `wasNotInvoked`

File: logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContentTest.kt [hunk 2]
```diff
@@ -136,17 +137,17 @@ class ClearConversationContentTest {
             verify(assetRepository)
                 .suspendFunction(assetRepository::deleteAsset)
                 .with(anything(), anything(), anything())
-                .wasInvoked()
+                .wasNotInvoked()
```
Comment (ohassine) [inline logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContentTest.kt:140]: same here

File: logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/backup/RestoreWebBackupUseCaseTest.kt (hunk 1 above)
Comment (MohamadJaara) [reply logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/backup/RestoreWebBackupUseCaseTest.kt:131]: the test is for unsupported Conversations, so the insert should not be inserted, and the `wasInvoked()` before the update did literally nothing.

File: logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContentTest.kt (hunk 2 above)
Comment (MohamadJaara) [reply logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContentTest.kt:140]: same, as above this test is if message delete failed so no assets are deleted

**Review by ohassine — APPROVED** [review]

**Pull request merged**